tetris.py

```diff
@@ -475,16 +482,21 @@ def flasher(self, full_lines, fill):
             for x_line in full_lines:
                 self.main_board.draw_rectangle(x, x_line, "flash", fill)
 
+    def get_time(self):
+        return time.time() - self.start_time - self.paused_time
+
     def timer(self):
         if self.status == GameStatus.in_progress:
-            game_time = time.time() - self.start_time - self.paused_time
-            self.topbar_time.config(text=f"{int(game_time / 60):02d}:{int(game_time % 60):02d}")
+            self.get_time()
```

Comment:
What is this line of code supposed to do? It calls the function without using the return value for anything.